src/Elmish.WPF/Binding.fs

```diff
@@ -254,6 +254,48 @@ module Binding =
       vopt bindings
       >> mapModel ValueSome
 
+  module SubModelT =
```

Comment:
I made these separate modules for sub-models that use view model base. I'm not sure how to organize them though. Got any preferences?

Reply:
@TysonMN

Reply:
I will have to look at this on my laptop

Reply:
We used this on our internal project, and it seemed to work well